lib/commands/general.js

```diff
@@ -44,5 +45,22 @@ commands.getWindowSize = async function (windowHandle = 'current') {
   return await this.executeAtom('get_window_size', []);
 };
 
+commands.getDeviceTime = iosCommands.general.getDeviceTime;
```

Comment:
does this mean I could have done this https://github.com/appium/appium-xcuitest-driver/blob/master/lib/commands/gesture.js#L42-L50 with just
`commands.isDrag = iosCommands.gestures.isDrag;` ?